.github/workflows/nightly.yml

```diff
@@ -0,0 +1,66 @@
+name: Nightly Build Packages
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete branch
+        uses: dawidd6/action-delete-branch@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branches: nightly
+
+      - name: Create Nightly Branch
+        uses: peterjgrainger/action-create-branch@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: 'nightly'
+
+      - name: Checkout nightly
+        uses: actions/checkout@v2
+        with:
+          ref: nightly
+      - name:  Activity check
+        run: |
+            curl -sL https://api.github.com/repos/${{github.repository}}/commits | jq -r '[.[]' > $HOME/commit.json
+            date="$(jq -r '.commit.author.date' $HOME/commit.json)"
+            timestamp=$(date --utc -d "$date" +%s)
+
+            days=$(( ( $(date --utc +%s) - $timestamp ) / 86400 ))
```

Comment:
BTW, the GitHub provides a parameter `since` for filtering the commits by date, just curious if there is any other reason to write the time filter in bash :)

Reply:
> BTW, the GitHub provides a parameter `since` for filtering the commits by date, just curious if there is any other reason to write the time filter in bash :)

https://api.github.com/repos/alibaba/pipcook/commits?since=yesterday works! It looks easier to read :p